app/views/internal/configs/show.html.erb

```diff
@@ -170,6 +170,15 @@
               <div class="alert alert-info">Used to determine what a member will be called e.g developer, hobbyist etc.</div>
             </div>
 
+            <div class="form-group">
+              <%= f.label :community_action, "Community action" %>
```

Comment:
Community Action sounded okay to me but I spent a lot of time on trying to get the appropriate name for the field, if anyone feels its terrible, let me know 😅